apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -49,10 +49,10 @@ const ThreeColumns = props => {
 
   return (
     <div style={{...defaultStyles.container, ...styles.container}}>
+      <div style={{...defaultStyles.left, ...styles.left}}>{children[0]}</div>
```

Comment:
@samantha-code - this is where I switch the order of the keyboard-navigation. I can easily switch it back so that the user navigates to all the chat bubbles (including instructions, hints, feedback) and then to the character/lightbulb.

Reply:
I think you should consider moving it to the end of the flow - thinking about when a user would want a hint and it would most likely be after they have been in the workspace trying something, so I think the less tabs between the workspace and the hint the better

Reply:
Sounds good! I updated code and after-update video. I agree after seeing it in action. Thanks!